tests/flask/test_upload.py

```diff
@@ -0,0 +1,39 @@
+import json
+from io import BytesIO
+
+import pytest
+
+from .app import create_app
+
+
+@pytest.fixture
+def flask_client():
+    app = create_app()
+
+    with app.test_client() as client:
+        yield client
```

Comment:
I think this is duplicated now :) you can move this fixture inside `tests/flask/conftest.py` to make it available in the flask tests